osbs/api.py

```diff
@@ -362,7 +363,7 @@ def _get_image_stream_info_for_build_request(self, build_request):
         image_stream_tag_name = None
 
         if build_request.has_ist_trigger():
-            image_stream_tag_id = build_request.spec.trigger_imagestreamtag.value
+            image_stream_tag_id = build_json['spec']['triggers'][0]['imageChange']['from']['name']
```

Comment:
Why this change?

Reply:
buildrequest2 has empty spec

Reply:
Good call! Maybe this should be a helper method in BuildRequest? BuildRequestV2 inherits from it so it should be available there too.